ansible/roles/products/hand_e/desktop-icons/tasks/main.yml

```diff
@@ -72,4 +72,94 @@
     window_title: shadow_save_log_app_xterm
     folder: "{{ save_latest_ros_logs_folder }}"
     shell_script_file_name: save_latest_ros_logs.sh
-    icon_file_name: log_icon.png
\ No newline at end of file
+    icon_file_name: log_icon.png
+
+- name: Copy Close Hand desktop icon
+  when: demo_icons
+  copy: 
+    src: files/close_hand_icon.png
+    dest: "{{ shadow_hand_launcher_folder }}/close_hand_icon.png"
+    mode: '664'
+
+- name: Create Close Hand executable launch script
+  when: demo_icons
+  template:
+    src: templates/scripts/rosrun_script.j2
+    dest: "{{ shadow_hand_launcher_folder }}/close_hand.sh"
+    mode: '755'
+  vars:
+    script_name: demo_rs.py
+
+- name: Create Close Hand desktop icon
+  when: demo_icons
+  template:
+    src: templates/desktop_labels/desktop_icon.j2
+    dest: "{{ user_folder }}/Desktop/Close_hand.desktop"
```

Comment:
why first word is capitilized and second not? I guess it is better if both are capitilized or lower case

Reply:
Fixed in the last commit